docset/winserver2019-ps/failoverclusters/New-Cluster.md

```diff
@@ -18,7 +18,7 @@ Creates a new failover cluster.
 ```
 New-Cluster [-Name] <String> [-Node <StringCollection>] [-StaticAddress <StringCollection>]
  [-IgnoreNetwork <StringCollection>] [-NoStorage] [-S2D] [-AdministrativeAccessPoint <AdminAccessPoint>]
- [-Force] [<CommonParameters>]
+ [-Force] [-ManagementPointNetworkType {Automatic | Singleton | Distributed}] [<CommonParameters>]
```

Comment:
```suggestion
 [-Force] [-ManagementPointNetworkType <AdminAccessPointResType>] [<CommonParameters>]
```

Reply:
At least as far as I can find examining the cmdlet in PowerShell, the defined parameter type is `object` so I don't think this change would be correct.

```powershell
(Get-Command New-Cluster).Parameters.ManagementPointNetworkType.ParameterType

IsPublic IsSerial Name                                     BaseType                                                     
-------- -------- ----                                     --------
True     True     Object                                                                                                                                                                             
```

Reply:
The parameter type is indeed AdminAccessPointResType according to PlatyPs:

![image](https://user-images.githubusercontent.com/36666927/126366362-2bb44afe-fcbb-48fc-9df2-c7fd79ad5b37.png)

Reply:
Let me know what you think @chasewilson  but we usually honor PlatyPs

Reply:
Yup usually the best thing to do. Good by me